spec/functions/number.md

```diff
@@ -413,6 +413,123 @@ contains an implementation-defined currency value
 of the _operand_ of the annotated _expression_,
 together with the resolved options' values.
 
+#### The `:percent` function
+
+> [!IMPORTANT]
+> The _function_ `:percent` has a status of **Draft**.
+> It is proposed for inclusion in a future release of this specification and is not Stable.
+
+The function `:percent` is a selector and formatter for percent values.
+
+##### Operands
+
+The function `:percent` requires a [Number Operand](#number-operands) as its _operand_.
+
+When either selecting or formatting the _expression_,
+the _resolved value_ of the _operand_ is multiplied by 100.
```

Comment:
This is being a little fast and loose with the term "resolved value", right? If the non-normative example is followed, the resolved value is an object with a bunch of methods, and `unwrap()` would have to be called to get the underlying number.

I'm not sure how I would phrase it better unless the underlying language about resolved values was made more precise, though.